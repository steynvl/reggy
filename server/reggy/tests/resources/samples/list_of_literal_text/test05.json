{
  "sampleStringsInfo": [
    [
      {
        "markerType": "List of literal text",
        "markedStrings": [
          "sa"
        ],
        "markerInfo": {
          "literalText": [
            "jannis",
            "kallis"
          ],
          "matchAnythingExceptSpecified": false,
          "caseInsensitive": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
