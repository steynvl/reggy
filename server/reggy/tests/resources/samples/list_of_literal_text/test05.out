Java
Pattern regex = Pattern.compile("(?:jann|kall)si");
(?:jann|kall)si
Perl
my $regex = /(?:jann|kall)si/;
(?:jann|kall)si
POSIX
(?:jann|kall)si
(?:jann|kall)si
Python
regex = re.compile(r'(?:jann|kall)si')
(?:jann|kall)si
JavaScript
const regex = /(?:jann|kall)si/;
(?:jann|kall)si
PHP
$regex = '/(?:jann|kall)si/';
(?:jann|kall)si
Golang
regex, _ := regexp.Compile("(?:jann|kall)si")
(?:jann|kall)si
Rust
let re = Regex::new(r"(?:jann|kall)si").unwrap();
(?:jann|kall)si
C#
var re = new Regex(@"(?:jann|kall)si");
(?:jann|kall)si
Scala
val re = "(?:jann|kall)si".r
(?:jann|kall)si
Kotlin
val regex = Regex("(?:jann|kall)si")
(?:jann|kall)si